2024-06-08 17:40:04,264 INFO    MainThread:356001 [wandb_setup.py:_flush():76] Current SDK version is 0.17.1
2024-06-08 17:40:04,264 INFO    MainThread:356001 [wandb_setup.py:_flush():76] Configure stats pid to 356001
2024-06-08 17:40:04,264 INFO    MainThread:356001 [wandb_setup.py:_flush():76] Loading settings from /home/yl3427/.config/wandb/settings
2024-06-08 17:40:04,264 INFO    MainThread:356001 [wandb_setup.py:_flush():76] Loading settings from /home/yl3427/cylab/selfCorrectionAgent/wandb/settings
2024-06-08 17:40:04,264 INFO    MainThread:356001 [wandb_setup.py:_flush():76] Loading settings from environment variables: {}
2024-06-08 17:40:04,264 INFO    MainThread:356001 [wandb_setup.py:_flush():76] Applying setup settings: {'_disable_service': False}
2024-06-08 17:40:04,264 INFO    MainThread:356001 [wandb_setup.py:_flush():76] Inferring run settings from compute environment: {'program_relpath': 'src/test_memory.py', 'program_abspath': '/home/yl3427/cylab/selfCorrectionAgent/src/test_memory.py', 'program': '/home/yl3427/cylab/selfCorrectionAgent/src/test_memory.py'}
2024-06-08 17:40:04,264 INFO    MainThread:356001 [wandb_setup.py:_flush():76] Applying login settings: {}
2024-06-08 17:40:04,264 INFO    MainThread:356001 [wandb_init.py:_log_setup():520] Logging user logs to /home/yl3427/cylab/selfCorrectionAgent/wandb/run-20240608_174004-ocewz339/logs/debug.log
2024-06-08 17:40:04,264 INFO    MainThread:356001 [wandb_init.py:_log_setup():521] Logging internal logs to /home/yl3427/cylab/selfCorrectionAgent/wandb/run-20240608_174004-ocewz339/logs/debug-internal.log
2024-06-08 17:40:04,264 INFO    MainThread:356001 [wandb_init.py:init():560] calling init triggers
2024-06-08 17:40:04,264 INFO    MainThread:356001 [wandb_init.py:init():567] wandb.init called with sweep_config: {}
config: {'class': 'ConditionalMemoryAgent', 'model': 'mistralai/Mixtral-8x7B-Instruct-v0.1', 'label': 't'}
2024-06-08 17:40:04,264 INFO    MainThread:356001 [wandb_init.py:init():605] wandb.init() called when a run is still active
2024-06-08 17:44:17,877 WARNING MsgRouterThr:356001 [router.py:message_loop():77] message_loop has been closed
